x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/utils.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SearchResponse, SearchHit, SignalHit } from '../../types';
+import { Logger } from '../../../../../../../../src/core/server';
+import { AlertServices } from '../../../../../alerting/server/types';
+
+export class ScrollAndBulkIndex {
+  service: AlertServices;
+  logger: Logger;
+  signalParams: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I have another PR which will fix this `any` issue here as in that PR I start a `types.ts` file and moved the first repeating types into that file.